src/components/PopupWithConfirmation.js

```diff
@@ -1,39 +1,38 @@
 import Popup from "./Popup.js";
-
 export default class PopupWithConfirmation extends Popup {
   constructor(popupSelector) {
-    super(popupSelector);
-    this._form = this._popup.querySelector(".modal__form");
-    this._submitButton = this._popup.querySelector('.modal__button');
+    super({ popupSelector });
```

Comment:
`  super({ popupSelector });`

* the app starts with errors https://disk.yandex.ru/i/P_1IT4a7HeAP0Q 

You shouldn't change the right code. Don't add new code if it's not needed because it takes another iteration to change it

You had this code here

```js

    super(popupSelector);

```


Please, bring it back